pkg/config/config.go

```diff
@@ -111,6 +132,8 @@ type V3ioConfig struct {
 	// Don't aggregate from raw chunks, for use when working as a Prometheus
 	// TSDB library
 	DisableClientAggr bool `json:"disableClientAggr,omitempty"`
+	// Build Info
+	BuildInfo *BuildInfo `json:"buildInfo,omitempty"`
```

Comment:
I think not only can we ditch the pointer, this could be a convenient place to use struct "inheritance" notation.

Reply:
e.g.:
```
type A struct {
  B
  extra uint64
}
```

Reply:
But not all that important.